libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx

```diff
@@ -457,31 +457,137 @@ export class SixItemPicker {
     return this.value;
   }
 
+  private handleKeyPress(event: KeyboardEvent) {
```

Comment:
`PageDown/Up`, and `Home/End`  are not firing a `six-item-picker-change` event

Reply:
I added the six-item-picker-change event in the changeValue method.